src/style.css

```
@@ -1 +1,73 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
```

```suggestion
html {
```
its better style fonts in `html` selector tag